src/components/Dropdown/Items/MultiSelectItem/MultiSelectItem.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-unnecessary-type-constraint */
 import React from 'react';
 import {
```

Comment:
It looks like maxWidth doesn't work. Label is not properly truncated:
![Screenshot 2024-11-08 at 11 30 47](https://github.com/user-attachments/assets/7538b0c5-f232-43e5-a94a-7b32b3430e73)

Reply:
I cannot reproduce it. Maybe you need to hard refresh your browser?
<img width="388" alt="Screenshot 2024-11-08 at 11 56 41" src="https://github.com/user-attachments/assets/ad5c3469-2fc0-4232-a59b-098acf6be6d0">

Reply:
try to change the pill name

Reply:
sorry... I see that you made the pill name long. I don't have a design for that. The design said that it is only the value that should be truncated. 
If you have a super long name, you can overwrite the `maxWidth` prop and make the pill bigger.

Reply:
I think it's not above the design. We have to handle it properly. Maybe,  at least have some maxWidth in percentages.

Reply:
I think that is up to Dima, as it is a new requirement. Can you ping him in the ticket with the question?

Reply:
ok